docs/releases/2.0.txt

```diff
@@ -282,6 +282,9 @@ Miscellaneous
   instead of dotted Python path strings. Django favors callable references
   since they provide better performance and debugging experience.
 
+* Calling `django.setup()` during a `django.setup()` call (reentrant call)
```

Comment:
I guess if the current behavior is the hang forever, then this isn't backwards incompatible considering that it has no chance to break working code. (side note: doouble backticks rather than single are required to have an effect on the rendered docs)